Firebase.cpp

```diff
@@ -15,6 +15,13 @@
 //
 #include "Firebase.h"
 
+// Detect whether stable version of HTTP library is installed instead of
+// master branch and patch in missing status and methods.
+#ifndef HTTP_CODE_TEMPORARY_REDIRECT
+#define HTTP_CODE_TEMPORARY_REDIRECT 307
+#define USE_STABLE_HTTPCLIENT_CORE
```

Comment:
maybe rename that into USE_ARDUINO_CORE_2_0_0 as what is "stable" is likely to change over time.